Ordering/Interfaces/IOrderManager.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Ordering.Common.Interfaces
+{
+    public interface IOrderManager
+    {
+        //This is used to add an item to an order
+        void AddItem(Object Item, int Quantity);
```

Comment:
This is an order manager so it is helpful to have a more "order specific" interface. Having an object as the input parameter here is too general.